src/components/Common/OnboardingStatusBadge/index.tsx

```diff
@@ -0,0 +1,20 @@
+import type { ContractorOnboardingStatus1 } from '@gusto/embedded-api/models/components/contractor'
```

Comment:
We can use this in EmployeeList as well. Not adding that to this PR as it's gone on long enough